<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_TextSettings</class>
 <widget class="QDialog" name="DLG_TextSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text</string>
  </property>
  <widget class="QPushButton" name="btn_bold">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_underlined">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>U</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_italic">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_fontSize">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>5</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
